source/includes/_authentication.md

```diff
@@ -1,10 +1,19 @@
 
 # Authentication
 
+To access the Apollo API, you need to authenticate your requests. There are two ways to do this: OAuth 2.0 and API keys.
+
+You should use OAuth 2.0 only if you are an Apollo Partner and want to utilize the Apollo API **on behalf of other Apollo users**. Otherwise, use API keys.
+
+You can request your OAuth 2.0 credentials and you API keys [in your API settings](https://app.apollo.io/#/settings/integrations/api).
```

Comment:
IMO "You can request your OAuth 2.0 credentials" => "You can register for Oauth 2.0"

Partner should register their app first to get credentials.

Reply:
Sure, here it is https://github.com/apolloio/apollo-api-docs/pull/77/commits/1abf3bc862717fb9a069b316e4d1deb21fe6c1c2